### PR TITLE
rename of holders and add some classes

### PR DESCRIPTION
**Pull request by Nigeriobike**

### CODE REVIEW

File: src/test/java/pageObjects/RegPageObj.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package pageObjects;
+
+import holders.DriverHolder;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class RegPageObj extends DriverHolder {
```
Comment (DereviankinNikita) [inline src/test/java/pageObjects/RegPageObj.java]: please rename the class using `elements` keyword to indicate that the elements are stored here

File: src/test/java/testNG/TestReg.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package testNG;
+
+import holders.TestInit;
+import org.testng.annotations.Test;
+
+public class TestReg extends TestInit {
```
Comment (DereviankinNikita) [inline src/test/java/testNG/TestReg.java]: The same for this class, name isn’t clear

**Review by DereviankinNikita — CHANGES_REQUESTED** [review]

File: src/test/java/testNG/TestReg.java (hunk 2 above)
Comment (Nigeriobike) [reply src/test/java/testNG/TestReg.java:6]: ok

**Comment by Nigeriobike** [comment]
maked

File: src/test/java/pageObjects/RegPageObj.java (hunk 1 above)
Comment (Nigeriobike) [reply src/test/java/pageObjects/RegPageObj.java]: maked

File: src/test/java/testNG/TestReg.java (hunk 2 above)
Comment (Nigeriobike) [reply src/test/java/testNG/TestReg.java]: Thats name of test , didnt understand

Comment (Nigeriobike) [reply src/test/java/testNG/TestReg.java]: Renamed

File: src/test/java/holders/TestInit.java [hunk 3]
```diff
@@ -10,15 +11,17 @@
...
 import java.time.Duration;
 
-public class TestInit {
-   public WebDriver driver;
-    Actions action ;
+public class TestInit extends Constructors {
+    public WebDriver driver;
+    public Actions action ;
 
-    @BeforeClass
```
Comment (DereviankinNikita) [inline src/test/java/holders/TestInit.java:21]: @Nigeriobike why did you decide to use before method instead before class?

File: src/test/java/holders/Constructors.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package holders;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+import pageObjects.HomePageElements;
+import pageObjects.HomePageHelper;
+import pageObjects.RegPageElements;
+import pageObjects.RegPageHelper;
+
+public class Constructors {
```
Comment (DereviankinNikita) [inline src/test/java/holders/Constructors.java]: @Nigeriobike What objects will used in tests, those in this class or in TestInit?

**Comment by Nigeriobike** [comment]
Made all

**Review by DereviankinNikita — APPROVED** [review]
Good job!

File: src/test/java/holders/TestInit.java (hunk 3 above)
Comment (Nigeriobike) [reply src/test/java/holders/TestInit.java:21]: That was check , forgot to delete

**Review by AnnaSokolenko — APPROVED** [review]

**Review by AnnaSokolenko — APPROVED** [review]

**Pull request merged**